src/config/config-schema.ts

```diff
@@ -194,6 +194,30 @@ const variantFieldSchema = Joi.custom((value, { error, state }) => {
     errors: { wrap: { label: false } }
 });
 
+const styleObjectModelReferenceError = 'styleObjectModelName.model.missing';
+const styleObjectModelNotObject = 'styleObjectModelName.model.type';
+const styleObjectModelNameSchema = Joi.string()
+    .allow('', null)
+    .custom((value, { error, state }) => {
+        const models = getModelsFromValidationState(state);
+        const modelNames = Object.keys(models);
+        const objectModelNames = modelNames.filter((modelName) => models[modelName]!.type === 'object');
+        if (!modelNames.includes(value)) {
+            return error(styleObjectModelReferenceError);
+        }
+        if (models[value]!.type !== 'object') {
```

Comment:
I think that the type should be "data" because the theme style object actually stored in a separate file